src/core/config/local_config.ts

```diff
@@ -0,0 +1,193 @@
+/**
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the ""License"");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an ""AS IS"" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import { IsEmail, IsNotEmpty, IsObject, IsString, validateSync } from 'class-validator'
+import { type ListrTask } from 'listr2'
+import fs from 'fs'
+import * as yaml from 'yaml'
+import { type ClusterMapping, type Deployment, type Deployments, type LocalConfigData } from './local_config_data.js'
+import { MissingArgumentError, SoloError } from '../errors.js'
+import { promptDeploymentClusters, promptDeploymentName, promptUserEmailAddress } from '../../commands/prompts.js'
+import { flags } from '../../commands/index.js'
```

Comment:
make sure that flags are imported prior to any other solo class, prevents circular dependency issues when running test cases individually or when running in `tsx` mode without compiled files.